app/Models/TaskComment.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class TaskComment extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'task_id',
+        'user_id',
+        'comment'
+    ];
+
+    public function task()
+    {
+        return $this->belongsTo(Task::class, 'task_id', 'id');
```

Comment:
```suggestion
        return $this->belongsTo(Task::class);
```
同様の箇所ほかにもあるので、確認してください。

Reply:
モデルのリレーションの引数を修正致しました。
ご確認よろしくお願い致します。
app/Models/TaskComment.php